ietf/static_src/css/iab.scss

```diff
@@ -0,0 +1,200 @@
+// Color swatches:
+$brand-blue: #3e96c8; // (62,150,200)
+$blue: #0b8cc5; // (11, 140, 97)
+$blue-2: #004f73; // (0, 79, 115)
+$navy: #002d3c; // (0, 45, 60)
+$yellow: #ffd13e; // (255, 209, 62)
+
+$grey-1: #f7fafc; // (247, 250, 252)
+$grey-2: #f0f6fa; // (240, 246, 250)
+$grey-3: #e6edf2; // (230, 237, 242)
+$grey-4: #d9e0e5; // (217, 224, 229)
+$grey-5: #cbd2d6; // (230, 210, 214)
+
+.h1-iab {
+    max-width: 682px;
+    line-height: 1.09;
+    font-size: 2.75rem;
+}
+
+.btn-secondary-iab {
+    background-color: $blue-2;
+    border-radius: 4px;
+    padding: 16px;
+
+    &:hover {
+        background-color: $navy;
+    }
+}
+
+.home_page__jumbotron-iab {
+    // copied
+    background-position: 50%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+    &:before {
+        // second color is $brand-blue with 0% opacity
+        background-image: linear-gradient(
+            180deg,
+            $brand-blue 17.96%,
+            rgba(62, 150, 200, 0) 100%
+        );
+        // copied
+        content: '';
+        bottom: 0;
+        left: 0;
+        pointer-events: none;
+        position: absolute;
+        right: 0;
+        top: 0;
+    }
+}
+
+.bg-brand-blue-iab {
+    --bs-bg-opacity: 1;
+    background-color: $brand-blue !important;
+}
+
+.bg-blue-2-iab {
+    --bs-bg-opacity: 1;
+    background-color: $blue-2 !important;
+}
+
+.bg-navy-iab {
+    --bs-bg-opacity: 1;
+    background-color: $navy !important;
+}
+
+.btn-brand-blue-navbar-iab {
+    background-color: $brand-blue;
+    border-radius: 4px;
+
+    &:hover {
+        background-color: $blue-2;
+    }
+}
+
+.bg-navy-iab {
+    --bs-bg-opacity: 1;
+    background-color: $navy !important;
+}
+
+.logo-text-iab {
+    max-width: 124px;
+    font-size: 15px;
+    line-height: 17px;
+}
+
+.navbar-text-iab {
+    font-size: 14px;
+    line-height: 16px;
+}
+
+.footer-link-iab {
+    border-radius: 4px;
+
+    &:hover {
+        background-color: $blue-2;
+    }
+}
+
+.iab-blog-post {
+    a {
+        color: $brand-blue !important;
+        &:hover {
+            color: $blue-2 !important;
+        }
+    }
+}
+
+.card-text-iab {
+    font-size: 16px;
+    line-height: 21px;
+    /* or 131% */
+    letter-spacing: 0.4px;
+    font-weight: 300;
+}
+
+.date-text-iab {
+    color: $navy;
+    font-size: 14px;
+    line-height: 18px;
+    /* identical to box height, or 129% */
+    letter-spacing: 0.4px;
```

Comment:
See other comments about rems for font-size and unitless for line heights. Applies to all the CSS.

Reply:
changed